phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -1451,7 +1454,35 @@ private TableDescriptor ensureTableCreated(byte[] physicalTableName, PTableType
         }
         return null; // will never make it here
     }
-    
+
+    private boolean isSplitPolicyUpdatedForTaskTable(
```

Comment:
Since this method actually also replaces the split policy, we should rename this method to indicate that.